### DISCUSSION TITLE
Livecheck unable to get version for new cask

### DISCUSSION THREAD

**Discussion by tauqua** [Homebrew/discussions#1604] [Tap maintenance and Homebrew development] [answered]
I'm trying to create a new cask and having trouble understanding why my livecheck isn't working. [Here](https://github.com/tauqua/homebrew-cask/blob/lightwright/Casks/lightwright.rb) is the code. Running `brew livecheck --debug lightwright` lists all the correct values, but then says `Error: lightwright: Unable to get versions`. My regex is nothing special and I've tested this manually with the regex and raw html of the page and it's captured just fine. Any thoughts?

**Comment by nandahkrishna** [comment, marked as answer]
That page is populated using JavaScript, and so you'll have to check this page instead: https://www.lightwright.com/api/web/public/download/getLinks. Also, you don't need a `strategy :page_match` as that's the default.

This should work:
```
livecheck do
  url "https://www.lightwright.com/api/web/public/download/getLinks"
  regex(/Lightwright_(\d+(?:\.\d+)*)\.dmg/i)
end
```

**Reply by tauqua** [reply]
Ah, thanks!

**Reply by nandahkrishna** [reply]
Just updated my suggestion with a modified regex as well.
